include/databaseFunctions.php

```diff
@@ -1,34 +1,24 @@
 <?php
 	include('config/config.php');
 	include('include/db_query.php');
```

Comment:
I don't think you want to include these files here. Any file in /include should stand on its own and never be called directly, so you shouldn't build the dependencies within these files.

Not sure if that makes sense, but you can see an example here: https://github.com/tilikang/coding_fellowship_2018/tree/sample-blog-no-db

Notice how I have a single "include_all.php" file that then includes all the other files in the include directory. This way, each actual web page (i.e. the files not in `/include`) can just include that one file and it'll have access to all the functions in every other file.